### PR TITLE
Carbon v1.3.0 update

### PR DESCRIPTION
**Pull request by shiauweitsung**
1. add TW/CN lauguage translate
2. add primary/secondary button hover style
3. import / create account ui update
4. fix import account phrase textarea style
5. use aminox is default network
6. update default networks without reinstall

### CODE REVIEW

File: app/_locales/zh_TW/messages.json [hunk 1]
```diff
@@ -873,6 +1048,12 @@
   "showHexDataDescription": {
```
Comment (mayone) [inline app/_locales/zh_TW/messages.json:1192]: Missing translation
```json
"show": {
  "message": "顯示"
}
```

File: app/_locales/zh_TW/messages.json [hunk 2]
```diff
@@ -337,6 +411,10 @@
   "endOfFlowMessage6": {
```
Comment (mayone) [inline app/_locales/zh_TW/messages.json:498]: I found out they had the wrong translation for `endOfFlowMessage4`...
Not 助記**祠**, should be 助記詞

File: app/_locales/zh_TW/messages.json (hunk 1 above)
Comment (shiauweitsung) [reply app/_locales/zh_TW/messages.json:1192]: fixed in 58912402ec705258789f6da9e2fabfad980e929b

File: app/_locales/zh_TW/messages.json (hunk 2 above)
Comment (shiauweitsung) [reply app/_locales/zh_TW/messages.json:498]: fixed in 9eec762a693cd82689f4fd8fce569a7635f95ddd

File: app/_locales/zh_TW/messages.json [hunk 3]
```diff
@@ -924,8 +1117,36 @@
   "stateLogsDescription": {
     "message": "狀態紀錄包含您的公開帳戶位址和已傳送的交易資訊"
   },
```
Comment (mayone) [inline app/_locales/zh_TW/messages.json:1257]:
```suggestion
  },
  "statusConnected": {
    "message": "已連接"
  },
  "statusNotConnected": {
    "message": "未連接"
  },
```

Comment (mayone) [reply app/_locales/zh_TW/messages.json:1257]: Maybe help me compare `zh_TW` with `zh_CN` to check is there any other translation missing

File: app/_locales/zh_TW/messages.json (hunk 2 above)
Comment (mayone) [reply app/_locales/zh_TW/messages.json:498]: @shiauweitsung  助記祠 not fixed yet

# 祠 -> 詞

Comment (shiauweitsung) [reply app/_locales/zh_TW/messages.json:498]: fixed in f04500849438b9be7913cc8c25717c1a37c3ba72

File: app/_locales/zh_TW/messages.json (hunk 3 above)
Comment (shiauweitsung) [reply app/_locales/zh_TW/messages.json:1257]: fixed in 09d78e84819a1f965a64c79fd7af387d93acd7dd

File: app/_locales/zh_TW/messages.json [hunk 4]
```diff
@@ -766,19 +902,19 @@
     "message": "重置帳戶將清除您的交易紀錄"
   },
```
Comment (mayone) [inline app/_locales/zh_TW/messages.json:1018]:
```suggestion
  },
  "restore": {
    "message": "恢復"
  },
```

File: app/_locales/zh_TW/messages.json [hunk 5]
```diff
@@ -855,6 +1114,9 @@
   "settings": {
```
Comment (mayone) [inline app/_locales/zh_TW/messages.json:1171]:
```suggestion
  "sendSpecifiedTokens": {
    "message": "發送 $1",
    "description": "Symbol of the specified token"
  },
  "sendTo": {
    "message": 發送至"
  },
  "sendingNativeAsset": {
    "message": "正在發送$1",
    "description": "$1 represents the native currency symbol for the current network (e.g. ETH or BNB)"
  },
  "settings": {
```

File: app/_locales/zh_TW/messages.json [hunk 6]
```diff
@@ -474,15 +617,35 @@
...
   },
+  "importTokens": {
+    "message": "導入代幣"
+  },
+  "importTokensCamelCase": {
+    "message": "導入代幣"
+  },
   "importWallet": {
     "message": "匯入錢包"
   },
```
Comment (mayone) [inline app/_locales/zh_TW/messages.json:676]:
```suggestion
  },
  "insufficientFundsForGas": {
    "message": "燃料資金不足"
  },
```

File: app/_locales/zh_TW/messages.json [hunk 7]
```diff
@@ -316,27 +446,37 @@
   "editContact": {
     "message": "編輯聯絡資訊"
   },
+  "editGasPriceTooltip": {
+    "message": "此網絡在提交交易時需要一個“燃料價格”欄。燃料價格是您將按單位支付的金額。"
+  },
```
Comment (mayone) [inline app/_locales/zh_TW/messages.json:469]:
```suggestion
  },
  "editGasSubTextAmountLabel": {
    "message": "最大金額：",
    "description": "This is meant to be used as the $1 substitution editGasSubTextAmount"
  },
  "editGasSubTextFeeLabel": {
    "message": "最高收費："
  },
```

File: app/_locales/zh_TW/messages.json (hunk 4 above)
Comment (shiauweitsung) [reply app/_locales/zh_TW/messages.json:1018]: fixed in https://github.com/Alpha-Carbon/wallet-extension/pull/21/commits/47dd284d4cacbca212317a5b0b499afa6a996e36

**Comment by mayone** [comment]
- @shiauweitsung please update package version to `1.3.0` then squash and merge

**Pull request merged**